vnext/Desktop/React.Windows.Desktop.vcxproj

```diff
@@ -81,7 +82,8 @@
         JSI_EXPORT=;
         %(PreprocessorDefinitions)
       </PreprocessorDefinitions>
-      <AdditionalOptions>%(AdditionalOptions) /Zc:strictStrings /bigobj</AdditionalOptions>
+      <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);"$(ReactNativeWindowsDir)Microsoft.ReactNative";"$(ReactNativeWindowsDir)build\$(Platform)\$(Configuration)\Microsoft.ReactNative\Generated Files"</AdditionalIncludeDirectories>
+      <AdditionalOptions>%(AdditionalOptions) /Zc:strictStrings /bigobj /showIncludes</AdditionalOptions>
```

Comment:
>/showIncludes [](start = 72, length = 13)

Please remove before merge

Reply:
Oops, I was reviewing a wrong version.

---
In reply to: [427443738](https://github.com/microsoft/react-native-windows/pull/4924#discussion_r427443738) [](ancestors = 427443738)